### PR TITLE
Adding FuncUI support

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
[FuncUI](https://avaloniacommunity.github.io/Avalonia.FuncUI.Docs/) is a great Avalonia wrapper for MVU paradigm of building apps in F#.

### CODE REVIEW

File: WebView.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29230.47
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.31911.260
```
Comment (WhiteBlackGoose) [inline WebView.sln:4]: VS does this automatically. If you want, I can revert it if you approve the rest

**Comment by WhiteBlackGoose** [comment]
I read, understood and accept the OutSystems CLA

**Comment by WhiteBlackGoose** [comment]
(converting to draft as I've [done something wrong](https://github.com/OutSystems/WebView/issues/240) )

**Comment by WhiteBlackGoose** [comment]
Now it seems to be working well

**Comment by joaompneves** [comment]
Hi, first of all thanks for contributing to this project, I really appreciate.

In order to maintain the simplicity and the maintenance effort I would propose that instead of including your PR here that we create a link in the readme pointing to your sample. Is that ok? Can you provide a link to a sample repo?

**Comment by joaompneves** [comment]
Here it is: https://github.com/OutSystems/WebView/pull/241/

**Comment by WhiteBlackGoose** [comment]
Hi.

The PR is not really a sample, it's mainly support for FuncUI.

**Comment by WhiteBlackGoose** [comment]
In fact, the main file of the PR is [this](https://github.com/OutSystems/WebView/blob/5fe1276374092a72ecc61d357ebb1cdbe928ca70/WebViewControl.Avalonia.FuncUI/Library.fs) tiny file wrapping WebView for FuncUI. The reason I'm proposing it for this repo is because when I myself needed this component (I'm a complete noob in GUI in general!) I spent some time trying to understand where to find it for FuncUI.

**Comment by joaompneves** [comment]
Ok either way ...I prefer referring to your repo instead of including it here

**Comment by WhiteBlackGoose** [comment]
Gotcha. Okay, will move to my repo then and will let you know about the link in your PR!

**Pull request closed without merging**